src/Models/User.php

```diff
@@ -3,6 +3,7 @@
 namespace EscolaLms\Consultations\Models;
 
 use EscolaLms\Auth\Models\User as AuthUser;
```

Comment:
Do you need AuthUser functionality? If not, then you can just extend CoreUser / most packages don't need to depend on Auth package